SPECS/perl-Test-LeakTrace/perl-Test-LeakTrace.spec

```diff
@@ -5,56 +5,61 @@
 %global with_valgrind 1
 %endif
 
-Name:		perl-Test-LeakTrace
-Summary:	Trace memory leaks
-Version:	0.16
-Release:	17%{?dist}
-License:	GPL+ or Artistic
+Summary:        Trace memory leaks
+Name:           perl-Test-LeakTrace
+Version:        0.16
+Release:        17%{?dist}
```

Comment:
18 ?

Reply:
Thanks, I missed this one. It's only a linting change, but will update the changelog nevertheless.